classes_assignment.js

```diff
@@ -0,0 +1,58 @@
+// 1.
+//   * Create a Human class that takes in a name and age.
+//   * Add the function `ageOneYear` that ages the human.
+//   * Add the function `eating`, that logs "mmm, mmm, mmm, I'm love'n it".
+//   * Create an instance of the Human class.
+//     * console log your humans age
+//     * call ageOneYear on your human
+//     * console log their age again.
+//     * call eating on your human.
+//
+class Human {
+  constructor(name, age){
+    this.name = name,
+    this.age = age
+  }
+
+  ageOneYear(num) {
+    return this.age++;
+  }
+
+  eating(str){
+    return `${Human.name}mmm, mmm, mmmm, I'm love'n it"`
+  }
+}
+
+let trey = new Human("Trey", "31")
+console.log(trey.name);
+trey.ageOneYear();
+console.log(trey.age);
+trey.eating()
+
```

Comment:
Small note: variable names that are classes should begin with an uppercase.
Great job with this assignment!